engineio/socket.py

```diff
@@ -41,8 +41,10 @@ def poll(self):
 
     def receive(self, pkt):
         """Receive packet from the client."""
+        packet_name = packet.packet_names[pkt.packet_type] \
+            if pkt.packet_type in packet.packet_names else ''
```

Comment:
This will always return an empty string. `pkt.packet_type` is an int, `packet.packet_names` is a list of strings.